File: WindowsApp2/Form1.vb
Public Class Form1
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Text = Settings1.Default.Title
        Label1.Text = Settings1.Default.test1
        Label2.Text = Settings1.Default.test2
        Label3.Text = Settings1.Default.test3
    End Sub
End Class

File: WinFormsApp1/Form1.cs
namespace WinFormsApp1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            label1.Text = Properties.Settings1.Default.test1;
            label2.Text = Properties.Settings1.Default.test2;
        }
    }
}